PIP-748 accessioning devops

p-sud opened the pull request.

p-sud commented:
The lint checks fail because I think the infrastructure changes should be reviewed independently. If I were to commit the other files this PR would get cluttered.

There are no usable tests yet, so circleci doesn't run pytest or codecov. Those will be added whenever the test suite gets set up.

ottojolanki commented on .pre-commit-config.yaml:
```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://gitlab.com/pycqa/flake8
```
I noticed that there is a hook for flake8 in the pre-commit repo github.com/pre-commit/pre-commit-hooks, so this could be removed and added to the lowest one.

ottojolanki commented on .pre-commit-config.yaml:
```diff
@@ -0,0 +1,41 @@
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.2.3
+    hooks:
+      - id: trailing-whitespace
```
You can put flake8 here. I assume these stay better up to date than "third party" ones.

p-sud replied:
I did in the new update, it seems to work fine.

p-sud commented:
> I can't figure out how to add a comment to a binary file, but looks like you accidentally pushed .ds_store in there. https://www.jeffgeerling.com/blogs/jeff-geerling/stop-letting-dsstore-slow-you

Good catch. I squashed, removed the DS_Store from the commit, and added it to the gitignore.
